TextFormatter.java

```diff
@@ -18,20 +19,21 @@ public static void main(String[] args) {
 
   // Konstruktor
   public TextFormatter(int maxLineLength) {
-  
-    for(int i = 0; i < (text.length()/maxLineLength); i++){
-
-//line 1
-     if(i == 0) {
-        temp = temp + (text.substring(i, maxLineLength) + "\n");
-      } else {
-       temp = temp + (text.substring(i*maxLineLength, i*maxLineLength+maxLineLength) + "\n");
-     }
+    StringTokenizer st = new StringTokenizer(text);
```

Comment:
Tokenizer ist an sich eine gute Idee, der Variablenname st ist allerdings nicht sprechend...
Und wieso wird bereits im Konstruktor der Text aufbereitet, wenn er "offiziell" erst in der Methode print übergeben wird? So ost der Formatter ja nur für diesen Text verwendbar!

Reply:
geloest